Dockerfile

```diff
@@ -2,12 +2,13 @@ FROM node:6-slim
 
 WORKDIR /usr/src/app
 
+RUN apt-get update \
+    && apt-get install -y jq python git build-essential --no-install-recommends
+
 # Keep the .git directory in order to properly report version
 COPY . /usr/src/app
 
-RUN apt-get update \
-    && apt-get install -y jq python git build-essential --no-install-recommends \
-    && npm install --production \
+RUN npm install --production \
```

Comment:
What is the reason for this change? Doesn't this add an extra layer to the image?

Reply:
I think @ssalaues mentioned that this would speed up the docker build process?

Reply:
Yeah it's a faster docker build because the apt update and apt install doesn't ever change so the the builds can use that cached layer. It's not at all critical to the PR but helped speed up the feedback loop during build and testing.

Reply:
Yeah extra layers actually are a good thing when placed strategically, not sure why they have a bad rep ;)